src/client/csfle/client_encryption.rs

```diff
@@ -234,8 +281,7 @@ impl ClientEncryption {
 }
 
 /// Options for initializing a new `ClientEncryption`.
-#[derive(Debug, Clone, TypedBuilder)]
```

Comment:
With the builder for the primary entrypoint, it seemed like it would be confusing and redundant to have a builder for the options as well (this also goes for `AutoEncryptionOptions`).